src/components/Display.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { display } from '../styles/Display.module.css';
+
+function Display({ result }) {
+  return <div className={display}>{result.slice(0, 19)}</div>;
+}
+
+Display.propTypes = {
+  result: PropTypes.string,
+};
+
+Display.defaultProps = {
```

Comment:
We don't need `defaultProps` for this. We just have to attach `isRequired` to `result` in the `propTypes` above.